modules/local/process/generate_proteins_from_vcf.nf

```diff
@@ -0,0 +1,41 @@
+// Import generic module functions
```

Comment:
Unfortunately the folder structure inside `modules` has changed: now you can remove the `process` directory and place all the module files and `functions.nf` files under `modules/local/`

Reply:
This has changed